msupply-foundation/open-msupply
# Store filter and status log entry on create

Fixes #3558

# 👩🏻‍💻 What does this PR do? 
 
Most items on the original issue are already fixed, so that leaves us:

- create a status log entry when creating an asset, either manually or via the import
  - the status should be `Functional`
  - a comment should say `Created`
- add a store filter to the list view, if running on central server

In addition, I noticed that the import is incorrectly failing to match codes due to the default pagination limit of 100 records. Since we only have 159 records I've bumped the limit to 1000 which should work for a while.

I wanted to have the text `Created` be translatable, so have used an existing entry. That required correcting the key, which was 'label.entered' as I thought it might be misleading later. It has added to the file count, sorry about that.

# 🧪 How has/should this change been tested? 

* Create an asset: the asset should have status 'functional' with one status log that has a comment of 'created'
* Import assets: check for the status logs
* Test the store filter which 
  * shows on central server 
  * does not show on remote ( though the filter option is available and you can manually enter the store via the querystring )

## 💌 Any notes for the reviewer?
